.decent_ci-Windows.yaml

```diff
@@ -5,6 +5,7 @@ compilers:
     release_only: true
     build_package_generator:
      - IFW
+     - ZIP
```

Comment:
Github Actions are now building release packages for both 32 and 64 bit windows in both IFW and ZIP format as well.  (Both linked to the correct bitness of Python 3.7)  This is fine though.